Update message_extras.md (#6831)
braze-inc/braze-docs: _docs/_user_guide/personalization_and_dynamic_content/liquid/advanced_filters/message_extras.md

#### Can I use message_extras with in-app messages?

> Support for `message_extras` in the in-app message channel is coming soon!

Reviewer: @davidbielik per our style guide, we avoid referring to future features in documentation:

> Avoid references to future features, or suggestions that something may be supported in the future.
> Don’t use words and phrases that anchor your writing to a point in time, as they make content become quickly outdated. Focus on how the product works right now, not on what has changed (except for time-focused content, such as in release notes).

Generally I would suggest not making this change until `message_extras` are supported, but if we must, I would suggest language like this:

```suggestion
Using`message_extras` in the in-app message channel is not supported at this time.
```

cc: @josh-mccrowell-braze @internetisaiah

Author: it says "avoid" does that mean never? even if we are actively building it?

Reviewer 2: That's a good note @davidbielik , and something maybe we can consider slightly rephrasing in the style guide to be more clear. Thanks!

But to answer your question, I think a general rule of thumb is that content should "sound like docs" instead of "sound like a blog." If you'd like to enhance Bre's update to add some more specifics, you could do something like:

```
Using`message_extras` in the in-app message channel is not supported at this time. For more information, see [Braze February Updates](www.example.com/some/random/blog).
```
